srcpkgs/mako/template

```diff
@@ -1,18 +1,19 @@
 # Template file for 'mako'
 pkgname=mako
-version=1.4.1
+version=1.5
 revision=1
 build_style=meson
-configure_args="-Dzsh-completions=true -Dapparmor=true -Dwerror=false"
+configure_args="-Dzsh-completions=true -Dwerror=false"
```

Comment:
Why remove apparmor?

Reply:
Looks like they dropped it
https://github.com/emersion/mako/pull/338

Reply:
Please install the file manually in `post_install` now, please.